README.rst

```diff
@@ -73,7 +73,7 @@ Once you have that, you can run the tests with::
 
 You can specify what test(s) to run::
 
-	S3TEST_CONF=your.conf ./virtualenv/bin/nosetests s3tests.functional.test_s3:test_object_acl_grant_public_read
+	S3TEST_CONF=your.conf ./virtualenv/bin/nosetests s3tests.functional.test_s3:test_bucket_acl_grant_userid_read
```

Comment:
But I'm not sure if it's what we want. The case here tells how to specify the test instead of all the tests. I will update the pr if everyone wants that :smiley: